frontend/lib/requests/favorite.ts

```
@@ -0,0 +1,69 @@
+    },
+    sort: ["createdAt:desc"],
+  };
+
+  return await fetchAPI<
+    {
+      id: number;
+      droplet: {
+        id: number;
+        name: string;
+        slug: string;
+        tags: any[];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)